### PR TITLE
Clear description & properties fields when no extensions listed

### LINKED ISSUE #350: Installed Extensions fields not cleared
When no installed extensions are shown for an extension point, the extensions dialog should clear the Description and Properties fields.

### PR DESCRIPTION
**Pull request by jaywha**
Fixes #350!

### Disclaimer on Temp UI changes
Made some temp changes to the WinForm dialog while working so it would resize on my Win 10 Pro PC using VS 2019 on a LG TV; so, resizing issues could have come from my DPI scaling of 200%.

I reverted everything back, but thought I should mention my issues with the DPI scaling just in case I neglected to revert something back in the form.

### CODE REVIEW

**Comment by CharliePoole** [comment]
Modified your "Fix" statement so GitHub will recognize it and close the issue.

File: src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs [hunk 1]
```diff
@@ -115,7 +115,7 @@ private void InitializeComponent()
             // 
             // extensionListView
             // 
-            this.extensionListView.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left)
+            this.extensionListView.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left)
```
Comment (immeraufdemhund) [inline src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs]: trailing white space that wasn't their before

File: src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs [hunk 2]
```diff
@@ -145,7 +145,7 @@ private void InitializeComponent()
             // 
             // extensionPointDescriptionTextBox
             // 
-            this.extensionPointDescriptionTextBox.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)
+            this.extensionPointDescriptionTextBox.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)
```
Comment (immeraufdemhund) [inline src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs]: trailing white space that wasn't their before

File: src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs [hunk 3]
```diff
@@ -220,13 +221,13 @@ private void InitializeComponent()
...
             this.label3.Name = "label3";
-            this.label3.Size = new System.Drawing.Size(54, 13);
+            this.label3.Size = new System.Drawing.Size(110, 25);
             this.label3.TabIndex = 5;
             this.label3.Text = "Properties";
             // 
             // extensionDescriptionTextBox
             // 
-            this.extensionDescriptionTextBox.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)
+            this.extensionDescriptionTextBox.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)
```
Comment (immeraufdemhund) [inline src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs]: trailing whitespace

File: src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs [hunk 4]
```diff
@@ -270,7 +271,7 @@ private void ExtensionDialog_Load(object sender, System.EventArgs e)
             {
                 _extensionPoints = new List<IExtensionPoint>(_extensionService.ExtensionPoints);
                 _extensions = new List<IExtensionNode>(_extensionService.Extensions);
-
+
```
Comment (immeraufdemhund) [inline src/TestCentric/testcentric.gui/Dialogs/ExtensionDialog.cs]: trailing whitespace

**Comment by CharliePoole** [comment]
@jaywha @immeraufdemhund Since there hasn't been a reply, I'll pulled this down and removed the unnecessary changes, including the UI spacing stuff. Changes are merged in PR #360

**Pull request closed without merging**